polymesh/deploy/docker-entrypoint.sh

```diff
@@ -8,13 +8,13 @@ _term() {
 trap _term SIGTERM
 
 # If NETWORK_CHAIN_ID is not set and we have an http endpoint to use, then curl the chain to fetch it (might need to retry)
-if [[ -n $NETWORK_ENDPOINT ]] && [[ -z $NETWORK_CHAIN_ID ]]
+if [[ -n $NETWORK_HTTP_ENDPOINT ]] && [[ -z $NETWORK_CHAIN_ID ]]
```

Comment:
Why are we changing the name ? In the docker-compose file I still see the name as NETWORK_ENDPOINT - https://github.com/PolymeshAssociation/polymesh-subquery-dictionary/blob/main/polymesh/docker-compose.yml#L24C7-L24C23

Reply:
NETWORK_ENDPOINT and NETWORK_HTTP_ENDPOINT are two different things.

The former is for the websocket address and later for HTTP endpoint.